### ISSUE TITLE
Consider adding Ag specific location classes

### ISSUE THREAD

**Issue by tdavis9** [OpenAgFunding/development#8]
Would we need to add additional options to the location class such as river basin, experimental stations, farmer fields, major fertilizer plant, extension office, irrigation equipment, etc.

**Comment by timgdavies** [comment]
Can you say more about the use-cases for this? Was this something that came up from multiple users?

**Comment by tdavis9** [comment]
@mbacou I think this came from you, I didn't get this in any of my consultations, so maybe you can speak to the demand for this?

**Comment by mbacou** [comment]
Yes, this came from our CG end-users. We had started to expand on an existing IATI codelist `<location-classs>` (I believe) https://github.com/harvestchoice/hc-crp/blob/master/vocabularies/cg_location_class.csv. This would not require any schema change AFAIK but is probably low priority. I was told then to look into existing USGS agricultural features.

**Comment by stevieflow** [comment]
For info the [GeographicLocationClass](http://iatistandard.org/202/codelists/GeographicLocationClass/) codelist is embedded - meaning it would be subject to decimal upgrade criteria

The IATI element that this is used in - <[location-class](http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/location/location-class/)> does not have a vocabulary attribute - meaning that any additional codes would have to be piloted via an extension , in the short term
